lib/components/programs-list/index.tsx

```diff
@@ -69,7 +73,7 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
         <Heading>{title}</Heading>
         <Text fontSize="lg">{description}</Text>
         <Link
-          href={registerLink}
+          href={recordingLink ?? registerLink}
```

Comment:
If `recordingLink` is an empty string then the button will say `REGISTER HERE` but it will link to the empty string because you're using the nullish coalescing operator and an empty string isn't nullish (I think). 

Maybe replace the `??` with `||` so if `recordingLink` is an empty string, it will link to the `registerLink` like it's supposed to.

Reply:
sounds good